python_backend/core/rag_config.py

```
@@ -0,0 +1,181 @@
+
+            # 경로 접근 권한 검증
+            if not self.faiss_index_path.exists():
+                logger.warning(f"FAISS 인덱스 경로가 존재하지 않습니다: {self.faiss_index_path}")
+
+            if not self.documents_path.exists():
+                logger.warning(f"문서 경로가 존재하지 않습니다: {self.documents_path}")
+
+        except Exception as e:
+            logger.error(f"설정 검증 실패: {e}")
+
+    @lru_cache(maxsize=1)
```

Ruff (0.13.1)

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)